packages/admin/admin/src/common/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,109 @@
+import { ChevronRight } from "@comet/admin-icons";
+import { type ComponentsOverrides, Typography } from "@mui/material";
+import { css, type Theme, useThemeProps } from "@mui/material/styles";
+import { Fragment, type ReactNode } from "react";
+
+import { createComponentSlot } from "../../helpers/createComponentSlot";
+import { type ThemedComponentBaseProps } from "../../helpers/ThemedComponentBaseProps";
+
+type BreadcrumbsClassKey = "root" | "breadcrumbsList" | "breadcrumbsItem" | "separator";
```

Comment:
Please fully remove the now unused slot


```suggestion
type BreadcrumbsClassKey = "root" | "breadcrumbsItem" | "separator";
```